wsd/DocumentBroker.cpp

```diff
@@ -2321,6 +2321,19 @@ bool DocumentBroker::isStorageOutdated() const
             << " and the last uploaded file was modified at " << lastModifiedTime << ", which are "
             << (currentModifiedTime == lastModifiedTime ? "identical" : "different"));
 
+#ifdef ENABLE_DEBUG
+    if (_storageManager.getLastUploadedFileModifiedTime() != _saveManager.getLastModifiedTime())
+    {
+        const std::chrono::steady_clock::time_point now = std::chrono::steady_clock::now();
+        LOG_ERR("StorageManager's lastModifiedTime ["
```

Comment:
This seems to fire for me everytime I open and close a document in nextcloud.

e.g.: StorageManager's lastModifiedTime [Wed Mar 19 20:14:08.765 2025 (6s 920ms ago)] doesn't match that of SaveManager's [Thu Jan 01 01:00:00.000 1970 (1742415255s 686ms ago)]. File lastModifiedTime: [Wed Mar 19 20:14:15.677 2025 (0s 8ms ago)]| wsd/DocumentBroker.cpp:2328

Reply:
Yes, this is fixed recently. The issue was `#ifdef` vs `#if`.